pkg/restapi/issuer/operation/operations.go

```diff
@@ -366,6 +371,23 @@ func (c *Operation) registerHandler() {
 	}
 }
 
+func (c *Operation) getProfile(w http.ResponseWriter, r *http.Request) (*Profile, bool) {
```

Comment:
It would be helpful to add a short comment above this function indicating that the bool return indicates that whether the operation succeeded, and if it didn't, that the response has already been made.